pkgs/development/compilers/llvm/common/compiler-rt/default.nix

```diff
@@ -62,6 +62,10 @@ stdenv.mkDerivation (finalAttrs: {
         + lib.optionalString (lib.versionAtLeast release_version "14") ''
           cp -r ${monorepoSrc}/cmake "$out"
         ''
+        # Needs "libcxx/utils/merge_archives.py"
+        + lib.optionalString (lib.versions.major release_version == "13") ''
+          cp -r ${monorepoSrc}/libcxx "$out"
+        ''
```

Comment:
Was this previously being injected by override or something? I’m not seeing it in this PR.

I’m dropping LLVM 12–17 this week, so I don’t think we need to spend time fixing existing issues with LLVM 13 (and it goes against making this zero rebuilds).

Reply:
I'm not sure, that's what confused me.

Reply:
Oh, now I know what it is. `llvmPackages.compiler-rt.src` was literally the entire Git repo shoved in and not this special one we generate here.